### PR TITLE
feat: remove content server check

### PR DESCRIPTION
**Pull request by nchamo**
We added a new healthcheck endpoint in https://github.com/decentraland/catalyst/pull/489, that will help our clients determine whether they should connect to a specific catalyst or not.

Since this new endpoint exists, we can remove the responsibility from the lighthouse

### CODE REVIEW

**Review by pablitar — COMMENTED** [review]
This will create a hard dependency with having to use the health check on explorer when selecting the realm. Have we done that already?

I'd suggest we only merge this when that is implemented there.

**Comment by nchamo** [comment]
> This will create a hard dependency with having to use the health check on explorer when selecting the realm. Have we done that already?
> 
> I'd suggest we only merge this when that is implemented there.

@jmoguilevsky is close to finishing it. I agree, this won't be merged until that change is merged on the explorer

**Review by pablitar — APPROVED** [review]
Nice work!

**Pull request merged**
